api/v1/qdrantcluster_types.go

```diff
@@ -15,8 +15,22 @@ const (
 )
 
 const (
-	// The Annotation key to trigger a restart.
+	// RestartedAtAnnotationKey is the annotation key to trigger a restart.
+	// The annotation should be placed on the QdrantCluster instance.
+	// The value should be a [RFC3339 formatted] date.
+	// If the value is updated it will retrigger the restart.
+	// The operator will not remove the annotation once restarted.
+	// For historical reasons the key doesn't start with `operator.qdrant.com/`
 	RestartedAtAnnotationKey = "restartedAt"
+	// RecreateNodeAnnotationKey is the annotation key to recreate a certain node.
+	// The annotation should be placed on the QdrantCluster instance.
+	// Multiple annotations can be added, the operator will handle them all.
```

Comment:
How can one add multiple annotations under the same key at the same time? K8s allows only one annotation to exists under a given key on a resource.